test/hexpm/accounts/user_test.exs

```diff
@@ -165,4 +165,30 @@ defmodule Hexpm.Accounts.UserTest do
       assert :error == Auth.password_auth("new_username", "ignore_this")
     end
   end
+
+  describe "update_security/2" do
+    test "enables tfa", %{user: user} do
+      changeset = User.update_security(user, %{tfa_enabled: true})
+      assert changeset.valid?
+      refute is_nil(changeset.changes.tfa)
+      refute is_nil(changeset.changes.tfa.changes.recovery_codes)
```

Comment:
nitpick: 

this is a personal preference so take it with a grain of salt, but someone once told me "don't test what something isn't, test what something is", and it stuck with me. (somewhat similar to whitelist vs blacklist approach.) In this case, instead of asserting that it isn't nil we could assert that it is a list.

that being said, since the next assertions checks the length, this one is redundant: if the list is nil the next assertion would have failed and so I'd remove this. For the same reason, I'd remove the assertion above (`refute is_nil(changeset.changes.tfa)`) too.